Flask-Integration/spellCheck.py

```diff
@@ -13,27 +13,32 @@
 
 def words(text): return re.findall(r'\w+', text)
 
-WORDS = Counter(words(open('bigText.txt').read()))
+WORDS_ENGLISH = Counter(words(open('bigText.txt', encoding="UTF-8").read()))
+WORDS_IRISH = Counter(words(open('legal.txt', encoding="UTF-8").read() + open('bible.txt', encoding="UTF-8").read() + open('news.txt', encoding="UTF-8").read() ))
```

Comment:
We may need to review the logic to switch languages.

Reply:
Absolutely, we need to evaluate how to change those global variables from the Flask integration